### DISCUSSION TITLE
Multiple charts with the same x-axes but different y-axes

### DISCUSSION THREAD

**Discussion by AlexStox** [highfestiva/finplot#378] [Q&A] [answered]
Hello and thank you for this great library. 
Let's imaging 2 cases: 
1. There are N charts, that share the same x-axes (dates), but different y-axes. 
![image](https://user-images.githubusercontent.com/55058601/199114977-0c60365e-adff-4215-84c4-c2290efbb533.png)

2. There are many charts like this example. 
![image](https://user-images.githubusercontent.com/55058601/199115090-444694da-7859-4791-9f20-13af14093be3.png)
this was taken from here - it's available in mplfinance which I don't like. 

For case 1 I tried this: 
```
fplt.candlestick_ochl(datasrc=df1[['open', 'close', 'high', 'low']], ax=ax1)
fplt.candlestick_ochl(datasrc=df2[['open', 'close', 'high', 'low']], ax=ax2)
fplt.show()
```
but ax2 as I understood is not the main chart and is always limited. 

Is there any way to do these cases in finplot?

**Comment by highfestiva** [comment]
I don't understand the question. In what way is the second chart limited? And more importantly: what are you trying to achieve that is not working as expected? If you want more charts laid out as a grid, you will have to do that manually using Qt. See examples for more info on that.

**Reply by AlexStox** [reply]
> I don't understand the question. In what way is the second chart limited? And more importantly: what are you trying to achieve that is not working as expected? If you want more charts laid out as a grid, you will have to do that manually using Qt. See examples for more info on that.

I've really tried to find it, I wouldn't write here. 
Could you please suggest which keywords I should try to find articles or examples?

**Reply by highfestiva** [reply]
https://github.com/highfestiva/finplot/tree/master/finplot/examples

**Reply by AlexStox** [reply]
well, unfortunately, there are no examples of that, but I found out that for multiple charts on one window: 

1. I can simply use one of the global variables located here https://github.com/highfestiva/finplot/wiki/Settings :) 
`fplt.top_graph_scale = 1`
Putting this will not make the top pane bigger than the others, so at least this solves my first case. 

2. Use this great example https://github.com/highfestiva/finplot/issues/56


However, if there is any example of the second case (many panes-charts in one window), please give a link where to see it

**Comment by highfestiva** [comment, marked as answer]
How about this: https://github.com/highfestiva/finplot/blob/master/finplot/examples/dockable.py?

**Reply by AlexStox** [reply]
wow, this is amazing!!! Thank you!
